### PR TITLE
Added test case for issue #19836

### LINKED ISSUE #19836: Sum is_convergent returns incorrect result
`Sum(4/(n + 2) - 5/(n + 1) + 1/n,(n,7,oo)).is_convergent()` returns `False`, but this series is convergent.

### PR DESCRIPTION
**Pull request by abhaydhiman**
#### References to other Issues or PRs

Fixes #19836 

#### Brief description of what is fixed or changed
Added a test case for a convergent series referring to issue number #19836  

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by oscarbenjamin** [comment]
I think it would be better to add this as a test with the others in `test_is_convergent` and a comment referencing the issue like in the comit where this was fixed  b777e46. Also it's better to use `is S.True` rather than `== True`.

**Comment by abhaydhiman** [comment]
So did I have to remove this particular test from test_limits.py

File: sympy/series/tests/test_limits.py [hunk 1]
```diff
@@ -870,4 +870,4 @@ def test_issue_19766():
 def test_issue_16708():
     m, vi = symbols('m vi', positive=True)
     B, ti, d = symbols('B ti d')
-    assert limit((B*ti*vi - sqrt(m)*sqrt(-2*B*d*vi + m*(vi)**2) + m*vi)/(B*vi), B, 0) == (d + ti*vi)/vi
+    assert limit((B*ti*vi - sqrt(m)*sqrt(-2*B*d*vi + m*(vi)**2) + m*vi)/(B*vi), B, 0) == (d + ti*vi)/vi
```
Comment (oscarbenjamin) [inline sympy/series/tests/test_limits.py:873]: It looks like you've deleted the newline at the end of this file. It needs to be added back

Comment (abhaydhiman) [reply sympy/series/tests/test_limits.py]: ok, done

Comment (abhaydhiman) [reply sympy/series/tests/test_limits.py]: well, but Travis CI still not accepting the code.

Comment (oscarbenjamin) [reply sympy/series/tests/test_limits.py]: The tests are still running

Comment (abhaydhiman) [reply sympy/series/tests/test_limits.py]: Do, I have to make some more changes in the code for this particular PR?

**Comment by oscarbenjamin** [comment]
I think that the commits here should be squashed. We don't need 5 commits for this change (it messes up things like git blame if other parts of the code are changed and then restored unnecessarily).

You can squash this with something like:
```
$ git reset HEAD~5
$ git add filename
$ git commit
$ git push --force
```

**Comment by oscarbenjamin** [comment]
Alternatively if squashing is too difficult you can just open a new PR making only this one change in one commit.

**Comment by abhaydhiman** [comment]
> I think that the commits here should be squashed. We don't need 5 commits for this change (it messes up things like git blame if other parts of the code are changed and then restored unnecessarily).
> 
> You can squash this with something like:
> 
> ```
> $ git reset HEAD~5
> $ git add filename
> $ git commit
> $ git push --force
> ```

ok, Sure!

**Comment by abhaydhiman** [comment]
I created another PR #19879 doing the same thing with only one commit.

**Pull request closed without merging**